### ISSUE TITLE
ios autoconnect doesnt work 

### ISSUE THREAD

**Issue by compvisiondeveloper** [pauldemarco/flutter_blue#928]
Hi,
ios auto connect doesn't work when I connect more than one devices. Could you help me for this? Is there any config in info.plist file.?

**Comment by DanielPiflaks** [comment]
Hi, 
Is auto connect works for you on iOS system **when only one device is connected**?
For me, it works only on Android but on iOS it just never works.

**Comment by boskokg** [comment]
I had the same issue. I wrote where is the bug and how to workaround it.

**Comment by compvisiondeveloper** [comment]
@DanielPiflaks yes doesn't work auto connection  when only one device is connected  .

**Comment by DanielPiflaks** [comment]
@compvisiondeveloper So it seems that auto connection just never works on iOS with flutter blue.
@boskokg For others to see your workaround in the other opened bug, here it is: https://github.com/pauldemarco/flutter_blue/issues/353

**Comment by compvisiondeveloper** [comment]
Hi @DanielPiflaks ;
  I overlooked it your solution , i guess. Thanks for reply.

**Comment by dhaval-parmar-1108** [comment]
Auto connect never works for IOS. Can't find any workaround yet.

**Comment by FurkanTskn** [comment]
Hi @dhaval-parmar-1108 
We called a timer periodic function (per 10 second) that try connect to BLE device on start.
it's only solution that we have.
